src/components/PDF/ApplicationForm.tsx

```diff
@@ -205,12 +237,13 @@ class ApplicationForm extends Component<Props, State> {
   render() {
     const {
       pdfApplication,
-      formQuestions,
+      title,
+      description,
+      fields,
       formAnswers,
       submitSuccessful,
       currentPage,
       numPages,
-      startDate,
       formError,
     } = this.state;
 
```

Comment:
alert.show('Successfully submitted ...', { type: 'success' }) <- something like this

Reply:
Done